# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-x11-paths.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-134.0.6998.45/build/config/compiler/BUILD.gn.vanilla	2025-03-08 08:54:36.391874275 +0100
+++ chromium-134.0.6998.45/build/config/compiler/BUILD.gn	2025-03-08 08:55:29.179236537 +0100
@@ -279,7 +279,7 @@
   cflags_objc = []
   cflags_objcc = []
   rustflags = []
-  ldflags = []
+  ldflags = [ "-L/usr/X11R7/lib", "-L/usr/X11R7/lib64" ]
   defines = []
   configs = []
 
